src/crab/ebpf_domain.cpp

```
@@ -1284,15 +1279,15 @@ void ebpf_domain_t::operator()(const Assume& s) {
     }
 }
 
-void ebpf_domain_t::operator()(const Undefined& a) {}
+void ebpf_transformer::operator()(const Undefined& a) {}
 
 // Simple truncation function usable with swap_endianness().
 template <class T>
 constexpr T truncate(T x) noexcept {
```

cppcheck

[style] The function 'truncate' is never used.

(unusedFunction)